### ISSUE TITLE
Create tiered architecture.

### ISSUE THREAD

**Issue by M4SSD35TRUCT10N** [M4SSD35TRUCT10N/SimplexERP#12]
Not only separate frontend and backend but separate any component to be deployed alone.

**Comment by khoathanglong** [comment]
Are you being working on this project alone?

**Comment by M4SSD35TRUCT10N** [comment]
Yes and only part time - literally only part time of my part time. I'm learning by doing. I have no JS background.
